migration-tool/src/main/resources/scripts/generate_upgrade_sdk_dependencies_recipe.py

```diff
@@ -55,6 +70,11 @@ def write_recipe_yml_file(service_mapping):
         write_recipe_metadata(f, version)
         write_bom_recipe(f, version)
         for s in service_mapping:
+            # edge case : v1 contains modules: cloudwatch AND cloudwatchmetrics, which both map to cloudwatch in v2
+            # (there is no cloudwatchmetrics module in v2)
+            # service_mapping maps cloudwatch to cloudwatchmetrics, so we'll write cloudwatch-cloudwatch manually
+            if (s == "cloudwatch"):
+                write_cloudwatch_recipe(f, version)
             write_recipe(f, s, service_mapping, version)
```

Comment:
no, we want to write both
- cloudwatchmetrics (v1) -> cloudwatch (v2)
and
- cloudwatch (v1) -> cloudwatch (v2)

since v1 has those 2 modules that map to the single cloudwatch module in v2
we could do it outside the loop, but this way its written in alphabetical order in the generated recipe